Lib/timeit.py

```diff
@@ -260,6 +266,7 @@ def main(args=None, *, _wrap_timer=None):
     if args is None:
         args = sys.argv[1:]
     import getopt
+    import _colorize
```

Comment:
```suggestion
    import _colorize
    colorize_errors = _colorize.can_colorize()

```

and then just use `colorize=colorize_errors` below

Reply:
just doing this...~ will batch commit thanks